pkg/render/intrusion_detection.go

```diff
@@ -789,6 +789,10 @@ func (c *intrusionDetectionComponent) webhooksControllerContainer() corev1.Conta
 			Name:  "LINSEED_TOKEN",
 			Value: GetLinseedTokenPath(c.cfg.ManagedCluster),
 		},
+		{
+			Name:  "LINSEED_CLUSTER",
```

Comment:
@bartolini I would suggest we do not introduce a new variable to set the cluster name. We are currently trying to move away from using this config map. For managed clusters, the managed cluster ID will be set by Voltron. I think you probably need to leave cluster name blank when using the Linseed client.